docker-compose-installed.yaml

```diff
@@ -47,15 +48,15 @@ services:
       - '3000:3000'
     depends_on:
       - validator
-      - rethinkdb
+      - rethink
     command: |
       bash -c "
         set -x &&
-        DB_HOST=rethinkdb npm run init &&
-        VALIDATOR_URL=tcp://validator:4004 DB_HOST=rethinkdb node index.js
+        DB_HOST=rethink npm run init &&
+        VALIDATOR_URL=tcp://validator:4004 DB_HOST=rethink node index.js
       "
 
-  rethinkdb:
+  rethink:
```

Comment:
By the logic of the commit message, shouldn't this be `supply-rethink`?

Reply:
It's an external component, not a part of the repo, analogous to the Sawtooth components. An argument could be made for `rethink-db` or whatever, but I think considering there will only ever be one rethink component, this is appropriate.